packages/ai/src/models/chat.ts

```diff
@@ -30,6 +30,12 @@ export type ChatSendOptions<TOptions = Record<string, any>> = {
    * stream chunk
    */
   readonly onChunk?: TextChunkHandler;
+
+  /**
+   * disable automatic function calling
+   * @default true
+   */
+  readonly enableAutomaticFunctionCalling?: boolean;
```

Comment:
nit: `autoFunctionCalling` possibly to shorten this var name, but I realize this is VERY subjective so non blocking feel free to ignore